tls/s2n_record_write.c

```diff
@@ -252,6 +256,9 @@ int s2n_record_writev(struct s2n_connection *conn, uint8_t content_type, const s
 
     /* Rewrite the length to be the actual fragment length */
     uint16_t actual_fragment_length = data_bytes_to_take + padding + extra;
+    if (is_tls13_record) {
+        actual_fragment_length += TLS13_CONTENT_TYPE;
```

Comment:
should that be  TLS13_CONTENT_TYPE_LENGTH then?

Reply:
yes, I'll rename it